docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md

```diff
@@ -1,6 +1,10 @@
 ---
 id: cstor
 title: Troubleshooting OpenEBS - cStor
+keywords:
+  - cStor
+  - cStor troubleshooting
```

Comment:
Add 
-OpenEBS
- OpenEBS community

Reply:
SEO keywords are the words and phrases that searchers enter into search engines and these help them land on the correct page. IMO, `OpenEBS community` will be the right keyword for only the community page.